docs/design-language.md

```diff
@@ -223,4 +223,33 @@ Panels come with an assortment of configurable navigation controls:
 
 ![panels](/img/panels/panel-welcome.jpg "Welcome Dialog Panel") ![panels](/img/panels/panel-feature.jpg "Features Panel")
 
+## Dialogs
+
+### Modal Dialogs
+
+Occasionally, there will be a need to display information, warn the user, or present controls in a manner that needs to utilize a more focused interface to the user.
+
+![tabs](/img/dialogs/dialog.png "Dialog")
+
+Examples of this include:
+
+-   Units under the measurement toolbar
+-   Polygon Style under the draw toolbar
+-   Share my Map
+-   Save and Load Project
+
+In all of the above examples, the information and controls is rendered above the rest of the app interface, drawing the users focus to the new information, as well as not altering the state of the app behind.
+
+Additional Information regarding modal alerts can be found in the **<a href="https://developers.geocortex.com/docs/web/tutorial-import-map-model/#create-a-custom-component-with-a-model" target="_blank">Developer Center Information</a>**.
+
+### Alerts
+
+Dialogs are the preferred method of presenting alerts to the user, since they allow information that is relevant to a state change, or problem, without navigating away, or impacting the information within the app. This can include, but are not limited to:
```

Comment:
I don't think there is a "preferred" way of presenting alerts to a user. It's context specific. But maybe you are correct and we should talk about notifications and alerts as similar related things. Explain when to use a notification and when to use an alert.

Reply:
I've updated the wordage some. Maybe we should break this up into notifications and alerts seperately.

Reply:
I don't see updated wordage here.

Reply:
I would remove the start of the sentence and just start the alerts section with, "Alert dialogs allow information that is relevant...".